pages/reference.md

```diff
@@ -10,14 +10,15 @@ To achieve this, it is important that the plugin is actually deployed on website
 The plugin can never replace serverside validation and doesn't intend to do so. Having both in place gives you the necessary security for your application, as well as improved usability.
 
 # Markup recommendations
-Each input has a label associated with it: The for-attribute of the label refers to the id-attribute of the input.
+Mandated:  A 'name' attribute is required for all input elements needing validation, and the plugin will not work without this.  A 'name' attribute must also be unique to the form, as this is how the plugin keeps track of all input elements. 
+
+Optionally:  Each input can have a label associated with it, where the 'for' attribute of the label refers to the 'id' attribute of the input.  It's also a common practice to have 'id' and 'name' attributes with the same value, although keep in mind that since this plugin does not use the 'id' attribute, this is not mandatory.
```

Comment:
>  I dont know if it is obvious to anyone using the plugin that a id has to be unique. I am ok not noting that though

My note about `id` was only carried over from existing documentation using very similar wording...

Existing docs:

>  Usually name and id attributes should have the same value.

My proposed:

>  It's also a common practice to have 'id' and 'name' attributes with the same value...

Perhaps it's not clear enough this refers to `id` and `name` be the *same as each other*.